app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java

```diff
@@ -1,29 +1,62 @@
 package com.support.android.designlibdemo.activities;
```

Comment:
I mean: 4
 4.Make user be able to update phone number. It's not being saved on parse for now